### PR TITLE
Remove `@AccessLogAnnotation` annotation, replace with built-in access log

### PR DESCRIPTION
**Pull request by kezhenxu94**
All http servers have already built in the access logs, we don't bother to replicate the features by ourselves, and clutter the annotations in the API controllers.

With the built in logs, users can:

- Modify the log format as they want, so that they can parse or analyze the logs in any system.
- Change the logs file location to wherever they like.
- Configure paths to ignore without rebuilding the package.

### CODE REVIEW

**Review by EricGao888 — APPROVED** [review]

File: dolphinscheduler-standalone-server/src/main/resources/application.yaml [hunk 1]
```diff
@@ -242,6 +242,9 @@ server:
     mime-types: text/html,text/xml,text/plain,text/css,text/javascript,application/javascript,application/json,application/xml
   jetty:
     max-http-form-post-size: 5000000
+    accesslog:
```
Comment (EricGao888) [inline dolphinscheduler-standalone-server/src/main/resources/application.yaml:245]: Will this configuration log `@RequestAttribute(value = Constants.SESSION_USER) User loginUser`? Seems there are some credentials in `User` object.

Comment (kezhenxu94) [reply dolphinscheduler-standalone-server/src/main/resources/application.yaml:245]: > Will this configuration log `@RequestAttribute(value = Constants.SESSION_USER) User loginUser`? Seems there are some credentials in `User` object.

No, it logs the http request parameters, not the controller method's arguments. Obviously credentials are not in the request parameters, they are in http headers

Comment (kezhenxu94) [reply dolphinscheduler-standalone-server/src/main/resources/application.yaml:245]: It also only logs query string, not request body

**Review by ruanwenjun — APPROVED** [review]
LGTM.
I tested on standalone, 
```
    accesslog:
      enabled: true
      custom-format: '%{client}a - %u %t "%r" %s %O %{ms}Tms'
```

it look like below
```
127.0.0.1 - - [24/Aug/2023:06:34:38 +0000] "GET /dolphinscheduler/projects/analysis/process-state-count?startDate=2023-08-24%2000%3A00%3A00&endDate=2023-08-24%2014%3A34%3A38&projectCode=0 HTTP/1.1" 200 230 15ms
127.0.0.1 - - [24/Aug/2023:06:34:38 +0000] "GET /dolphinscheduler/projects/analysis/task-state-count?startDate=2023-08-24%2000%3A00%3A00&endDate=2023-08-24%2014%3A34%3A38&projectCode=0 HTTP/1.1" 200 236 15ms
127.0.0.1 - - [24/Aug/2023:06:34:38 +0000] "GET /dolphinscheduler/projects/analysis/define-user-count?projectCode=0 HTTP/1.1" 200 103 16ms
```

Can we log the username ? it seems `%u` doesn't work.

**Comment by kezhenxu94** [comment]
> LGTM. I tested on standalone,
> 
> ```
>     accesslog:
>       enabled: true
>       custom-format: '%{client}a - %u %t "%r" %s %O %{ms}Tms'
> ```
> 
> it look like below
> 
> ```
> 127.0.0.1 - - [24/Aug/2023:06:34:38 +0000] "GET /dolphinscheduler/projects/analysis/process-state-count?startDate=2023-08-24%2000%3A00%3A00&endDate=2023-08-24%2014%3A34%3A38&projectCode=0 HTTP/1.1" 200 230 15ms
> 127.0.0.1 - - [24/Aug/2023:06:34:38 +0000] "GET /dolphinscheduler/projects/analysis/task-state-count?startDate=2023-08-24%2000%3A00%3A00&endDate=2023-08-24%2014%3A34%3A38&projectCode=0 HTTP/1.1" 200 236 15ms
> 127.0.0.1 - - [24/Aug/2023:06:34:38 +0000] "GET /dolphinscheduler/projects/analysis/define-user-count?projectCode=0 HTTP/1.1" 200 103 16ms
> ```
> 
> Can we log the username ? it seems `%u` doesn't work.

The `%u` doesn't work because the authentication is not done as in Spring way, I shall fix these in following PRs

**Comment by kezhenxu94** [comment]
Merging, will look into the API-Test CI issue, might be something configured wrong

**Pull request merged**